awx/ui/src/constants.js

```diff
@@ -12,3 +14,18 @@ export const SESSION_TIMEOUT_KEY = 'awx-session-timeout';
 export const SESSION_REDIRECT_URL = 'awx-redirect-url';
 export const PERSISTENT_FILTER_KEY = 'awx-persistent-filter';
 export const SESSION_USER_ID = 'awx-session-user-id';
+
+export const VERBOSITY = {
```

Comment:
Good example of standardization here. If you wanted to just use one constant to represent Verbosity and it's permutations in the UI, I could see creating the object literal containing the `key`, `value`, and `label` keys and then calling a function to return the appropriate label when not used as an options dropdown.

```javascript
export const getVerbosity = (key) => VERBOSITY.filter(v => v.key === key)
const [{ label }] = getVerbosity(verbosityKey);
```

You could probably just save both the constant and the function as one object literal itself.